share/scripts/inverse/csg_table

```diff
@@ -96,6 +96,10 @@ postadd overwrite postadd_overwrite.sh
 postadd plot postadd_plot.sh
 postadd average postadd_average.sh
 
+# smoothing for updated potential to allow forgetting noise from previous iterations (rdfs)
+# the script is the same as postupd_smooth.sh
+postadd smooth postadd_smooth.sh
```

Comment:
Just undo them here (on this branch) and then put them on a different branch (starting from `master`).

However, maybe we should discuss the use case for `postupd smooth` on the mailing list (or in an issue) first, because I still don't see the need for it.

@symashayak: any opinion on that?